plugins/embed/gui-themes/themes/zeit-solo/item/base.dust

```diff
@@ -3,26 +3,16 @@
 {! Attention! Opening tag in headerContainer … !}
 {<headerContainer}
 	<div class="liveblog-post">
-		<div class="post-header">
```

Comment:
I just deleted this unused Element plus the corresponding CSS `.post-header { display: none; }`. I'm not sure if I understand the question right?